wp-content/themes/core/assets/css/src/admin/block-editor/editor.pcss

```diff
@@ -15,19 +15,20 @@
 .block-editor-block-list__layout {
 	/* Unset WP Core's vertical spacing on some headings, paragraphs, etc. */
 	* {
-		margin-top: 0;
-		margin-bottom: 0;
-	}
-
-	/* CASE: Alignment: Full */
-	.wp-block[data-align="full"] {
-		margin-left: auto;
-		margin-right: auto;
+		margin: 0;
```

Comment:
WordPress styles the full `margin` property in `common.css`. Changing from updating `margin-top` and `margin-bottom` allowed me to overwrite the core styles more easily and remove some extra css. I was also able to remove the now redundant `figure` styles on line 29.